### ISSUE TITLE
Details about Observation Normalization

### ISSUE THREAD

**Issue by CVHvn** [asyua-ye/Mymarios#6]
Hi, I see you normalize observation feature map (output of the last CNN layer, after fed raw image to cnn) instead of normalize raw image. Have you tested the impacts of this change?

**Comment by asyua-ye** [comment]
I'm sorry, I have not experimented with the impact of using state normalization alone. I experimented with the impact of using these three methods together (state normalization, learning rate decay, and reward scaling) and found that under the same hyperparameters, compared to PPO without these three methods, it converges faster and more stably.

The reason why I don't process raw images is because I realized that I have already processed them during environment preprocessing. Then, I treat the vectors obtained after CNN processing as the state. From this perspective, I thought about normalizing this variable.

**Comment by CVHvn** [comment]
Thank for your answer :D
